tests/libfuzzer/python_fuzzer.cc

```diff
@@ -0,0 +1,73 @@
+/*#include <stdint.h>
+#include <stdio.h>
+#include <string.h>*/
+
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <unistd.h>
+#include <ctype.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <errno.h>
+
+extern "C" {
+#include "py/nlr.h"
+#include "py/compile.h"
+#include "py/runtime.h"
+#include "py/repl.h"
+#include "py/gc.h"
+#include "py/obj.h"
+#include "lib/utils/pyexec.h"
+#include "py/stackctrl.h"
+}
+
+
+// modelled after micropython/minimal/main.c
+// and micropython/unix/main.c
+extern "C" int LLVMFuzzerTestOneInput(const unsigned char *data, size_t size) {    
+    
+    //Initialize stuff (?)
+    mp_stack_ctrl_init();
+
+    mp_stack_set_limit(40000 * (BYTES_PER_WORD / 4));
+
+    #if MICROPY_ENABLE_GC
+    // Heap size of GC heap (if enabled)
+    // Make it larger on a 64 bit machine, because pointers are larger.
+    long heap_size = 1024*1024 * (sizeof(mp_uint_t) / 4);
+    char *heap = (char*)malloc(heap_size);
```

Comment:
Unless you want to change the heap size on the fly, it's simpler to just create a static buffer (stored in the BSS, eg static uint8_t heap[1024*1024]) and use that for the heap.